services/production-tools/values-idfint.yaml

```diff
@@ -0,0 +1,13 @@
+production-tools:
+  butler_secret_path: "secret/k8s_operator/data-int.lsst.cloud/butler-secret"
+  environment:
+    BUTLER_URI: "s3://butler-us-central1-panda-dev/dc2/butler-external.yaml"
+    PGPASSFILE: "/home/worker/.lsst/postgres-credentials.txt"
+    AWS_SHARED_CREDENTIALS_FILE: "/home/worker/.lsst/aws-credentials.ini"
+    S3_ENDPOINT_URL: "https://storage.googleapis.com"
+    LOG_BUCKET: "drp-us-central1-logging"
+    LOG_PREFIX: "Panda-RubinLog"
+  ingress:
+    host: "data-int.lsst.cloud"
+    path: "/production-tools"
+    gafaelfawrAuthQuery: "scope=exec:portal&delegate_to=productiontools"
```

Comment:
Do you need delegated Gafaelfawr tokens for something? That's only required if you need to make authenticated calls on behalf of the user to other services. (If you do need this, use `production-tools` to match the name of the service.)

Also, this should be set in `values.yaml`, not here, since it won't vary by environment.